docs/drivers/aws.md

```diff
@@ -152,6 +152,14 @@ To create a machine with a non-default vpc-id:
 
 This example assumes the VPC ID was found in the `a` availability zone. Use the`--amazonec2-zone` flag to specify a zone other than the `a` zone. For example, `--amazonec2-zone c` signifies `us-east1-c`.
 
+## VPC Connectivity
+Machine uses ssh to complete the set up of instances in EC2 and requires the ability to access the instance directly.  
+
+If you use the flag `--amazonec2-private-address-only`, you will need to ensure that you have some method of accessing the new instance from within the internal network of the VPC (e.g. a corporate VPN to the VPC, a VPN instance inside the VPC or using Docker-machine from an instance within your VPC). 
+
+## VPC Set up
+Configuration of VPCs is beyond the scope of this guide, however the first step in troubleshooting is ensuring if you are using private subnets that you follow best practices and have some form of NAT available so that the set up process can access the internet to complete set up.
```

Comment:
> Configuration of VPCs is beyond the scope of this guide. However, it's worth noting that if you are using private subnets, you follow best practices _(link?)_ and have NAT available so that the setup process can access the Internet.